src/content/docs/setup/configuration/gated-content.mdx

```diff
@@ -34,7 +34,7 @@ The following diagram illustrates the workflow when gated content is enabled.
 ![Gated content workflow](@images/gated-content.svg)
 </Diagram>
 
-The shopper requests a page on the storefront. The request is sent to the CDN (Fastly in this case), which then routes it to Edge Delivery Services. Edge Delivery Services returns the requested document with headers defined in the `header` or `header.xlsx` file. Fastly Compute checks whether the requested page is considered gated content based on the headers returned by Edge Delivery Services or by comparing it with a hardcoded list of gated content. If the header defined in the `header` or `header.xlsx` file is not present, or if the requested URL is not in the hardcoded gated content list, the request is processed normally.
+The shopper requests a page on the storefront. The request is sent to the CDN (Fastly in this case), which then routes it to Edge Delivery Services. Edge Delivery Services returns the requested document with headers defined in the `header` document. Fastly Compute checks whether the requested page is considered gated content based on the headers returned by Edge Delivery Services or by comparing it with a hardcoded list of gated content. If the header defined in the `header` document is not present, or if the requested URL is not in the hardcoded gated content list, the request is processed normally.
```

Comment:
When you have time, it would be good to make this bulleted so it's easier to scan. I know it's explaining the graphic above, but still would be easier to consume as chunked info.